testing/trino-tests/src/test/java/io/trino/tests/AbstractTestEngineOnlyQueries.java

```diff
@@ -577,7 +582,6 @@ public void testAssignUniqueId()
     @Test
     public void testAtTimeZone()
     {
-        // TODO the expected values here are non-sensical due to https://github.com/trinodb/trino/issues/37
```

Comment:
The semantics of these values are covered extensively by https://github.com/trinodb/trino/blob/master/core/trino-main/src/test/java/io/trino/operator/scalar/timestamp/TestTimestamp.java#L2639